Artifacts/windows-azuredevops-multiple-agents/artifactfile.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "Azure Pipelines Agent(s)",
+    "publisher": "Utkarsh Shigihalli",
+    "description": "Downloads latest Azure Pipelines agent for the account, and installs the specified number instances of the agents inside one VM. Registers the agent with the Azure DevOps account, and adds agents to the specified agent pool. Note - If the specified agent folder already exists it will be overwritten.",
+    "tags": [
+      "Azure DevOps",
+      "Pipelines",
+      "Release",
+      "Build",
+      "CI",
+      "Windows"
+    ],
+    "iconUri": "https://cdn.vsassets.io/content/icons/favicon.ico",
+    "targetOsType": "Windows",
+    "parameters": {
+      "Account": {
+        "type": "string",
+        "displayName": "Azure DevOps Account Name",
+        "description": "The name of the Azure DevOps account to add the build agent to. This is the value in your Azure DevOps URL: e.g. 'myaccount' in https://dev.azure.com/myaccount."
+      },
+      "PersonalAccessToken": {
+        "type": "securestring",
+        "displayName": "Azure DevOps Personal Access Token",
+        "description": "A personal access token with permissions to add build agents. It will only be used to register the agent."
+      },
+      "AgentName": {
+        "type": "string",
+        "displayName": "Agent Name",
+        "description": "The name to give to the agent, as seen by Azure DevOps. If empty, the computer name will be used. Prefixes every agent with given prefix",
+        "allowEmpty": true
+      },
+      "AgentNamePrefix": {
+        "type": "string",
+        "displayName": "Agent Name Prefix",
+        "description": "Prefixes every agent with string specified here",
+        "allowEmpty": true
+      },
+      "PoolName": {
+        "type": "string",
+        "displayName": "Agent Pool",
+        "description": "The agent pool this build agent should be added to."
+      },
+      "AgentCount":{
```

Comment:
Consider setting the default to `1` and make this mandatory to be more explicit to the user. I do see in the PS1 logic that we do `$AgentCount + 1` in the `for` loop, which will give `1` if not set. So, the logic will still work. Food for thought.

Reply:
Agree. Will make the change